chef_master/source/resource_mount.rst

```diff
@@ -239,9 +239,9 @@ This resource has the following properties:
    .. end_tag
 
 ``supports``
-   **Ruby Type:** Array
+   **Ruby Type:** Hash
 
-   An array of options for supported mount features. Default value: ``:remount``.
+   Specify a Hash of supported mount features. Default value: ``remount: true``.
```

Comment:
Looking at the code and the comment above state that the default value, when not specifying a Hash of values, is `remount: false`.